.gitignore

```diff
@@ -1,5 +1,7 @@
 # Hand-added
 build
+.replit
```

Comment:
Thanks. That will help working with replits with custom run commands. Will merge momentarily @piam